Sources/CYaml/BUILD

```diff
@@ -0,0 +1,16 @@
+load(
+    "@rules_cc//cc:defs.bzl",
+    "objc_library"
+)
+
+objc_library(
```

Comment:
Does `cc_library` work? `objc_library` doesn't support Linux, while `Yams` does.

Reply:
I believe it would, I will give it a try